src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -62,12 +62,13 @@ def __init__(self, name, location, datasetpath):
         self.location     = location
         self.datasetpath  = datasetpath
         self.workflows    = set()
+        self.uploader      = uploader
```

Comment:
Even though the previous lines have this indented assignment, could you please surround the assignment by a single spacing, e.g.: `        self.uploader = uploader`